### DISCUSSION TITLE
I received a critical error while trying to launch the game

### DISCUSSION THREAD

**Discussion by rafciu41** [Master-64/Shrek-2.2#3] [General]
No Label

OS: Windows NT 6.2 (Build: 9200)
CPU: GenuineIntel PentiumPro-class processor @ 2904 MHz with 4095MB RAM
Video: NVIDIA GeForce RTX 3060 (1)

Failed to enter Entry: Can't find file for package 'AllDialog'

History: UGameEngine::Init <- InitEngine

**Comment by Mirror12321** [comment]
you need to unzip AllDialog.rar and extract AllDialogue.uax to your Sounds folder

**Comment by Master-64** [comment]
You must unzip AllDialog.7z.001 with 7-Zip as explained in the installation readme.
